distribution/release-seata.xml

```diff
@@ -42,20 +42,18 @@
             </includes>
             <fileMode>0755</fileMode>
         </fileSet>
+
+        <fileSet>
+            <includes>
+                <include>lib/*</include>
+            </includes>
+        </fileSet>
     </fileSets>
     <files>
         <file>
             <source>LICENSE-BIN</source>
             <destName>LICENSE</destName>
         </file>
-        <file>
-            <source>NOTICE-BIN</source>
-            <destName>NOTICE</destName>
-        </file>
-        <file>
-            <source>../server/target/seata-server-${project.version}.jar</source>
```

Comment:
when mvn install，it will create a xxx.tar.gz?

Reply:
My test result.

![image](https://user-images.githubusercontent.com/48389485/57603207-aa574880-7593-11e9-9e76-d55190d14eb7.png)